cmd/bblfsh/client.go

```diff
@@ -15,6 +15,11 @@ import (
 	"github.com/Sirupsen/logrus"
 	"github.com/bblfsh/sdk/protocol"
 	"google.golang.org/grpc"
+	"srcd.works/go-errors.v0"
+)
+
+var (
+	errProfiler = errors.NewKind("Failed to create a % file at %s")
```

Comment:
I did not export this var intentionally.
Do you see any cases for this error to be useful outside of the current package?